### PR TITLE
Add details logic and view

### PR DESCRIPTION
**Pull request by PiotrChlapinskiBinarapps**
Now user has more information about weather in details view like weather forecast for next days and hours.

### CODE REVIEW

File: Weather App/WeathersList/view/DayForecastWeatherCell.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    private var dayLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .custom(.white)
+        label.font.withSize(10)
+        return label
+    }()
+
+    private var temperatureLabel: UILabel = {
+        let label = UILabel()
+        label.text = "12 C"
```
Comment (jarekdominiak) [inline Weather App/WeathersList/view/DayForecastWeatherCell.swift]: I suppose to, default value should have `°C`

File: Weather App/WeathersList/view/DayForecastWeatherCell.swift [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+        mainStack.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(5)
+            make.top.bottom.equalToSuperview().inset(10)
+        }
+    }
+
+    public func setup(weatherForecast: OneForecast) {
+        dayLabel.text = "\(weatherForecast.date.dayAndMonth)"
+        temperatureLabel.text = "\(weatherForecast.mesurements.temperature.int)°C"
```
Comment (jarekdominiak) [inline Weather App/WeathersList/view/DayForecastWeatherCell.swift]: to display temperature, you can use `MeasurementFormatter`

```
let formatter = MeasurementFormatter()
formatter.numberFormatter.maximumFractionDigits = 0
let measurement = Measurement(value: 25, unit: UnitTemperature.celsius)
print("temprature \(formatter.string(from: measurement))")
```

File: Weather App/WeathersList/viewModel/CityWeatherViewModel.swift [hunk 3]
```diff
@@ -19,22 +19,29 @@ public protocol CityWeatherViewModel {
...
     public var weatherForLocation: CityWeather?
     private let citiesRepository: CitiesRepository
+    private let forecastRepository: ForecastRepositoryImpl
+
     private let storage: Storage
     private let delegate: WeathersListViewControllerDelegate
     private var cities: Cities
 
-    init(delegate: WeathersListViewControllerDelegate, storage: Storage = StorageUserDefaults(), citiesRepository: CitiesRepository = CitiesRepository()) {
+    init(delegate: WeathersListViewControllerDelegate, storage: Storage = StorageUserDefaults(), citiesRepository: CitiesRepository = CitiesRepository(), forecastRepository: ForecastRepositoryImpl = ForecastRepositoryImpl()) {
```
Comment (jarekdominiak) [inline Weather App/WeathersList/viewModel/CityWeatherViewModel.swift]: I think, one repository for all requests will be enough. You can rename old one to `WeatherRepository`

File: Weather App/WeathersList/controller/WeatherDetailsViewController.swift [hunk 4]
```diff
@@ -1,23 +1,62 @@
...
+
+    private func configureView() {
         cityNameLabel.text = weather.name
         temperatureLabel.text = "\(Int(weather.mesurements.temperature)) °C"
-        pressureLabel.text = "\(weather.mesurements.pressure) hPa"
-        humidityLabel.text = "\(weather.mesurements.humidity) %"
-        weatherDescriptionLabel.text = "\(weather.description[0].descriptionWeather)"
+        humadityLabel.text = "\(weather.mesurements.humidity)%"
+        pressureLabel.text = "\(weather.mesurements.pressure)hPa"
+        windSpeedLabel.text = "\(weather.wind.velocity)km/h"
```
Comment (jarekdominiak) [inline Weather App/WeathersList/controller/WeatherDetailsViewController.swift]: I think you can use formatters for all measurements, I know, you can use `UnitPressure` https://developer.apple.com/documentation/foundation/unitpressure

File: Weather App/WeathersList/view/DayForecastWeatherCell.swift (hunk 1 above)
Comment (PiotrChlapinskiBinarapps) [reply Weather App/WeathersList/view/DayForecastWeatherCell.swift]: 👍

File: Weather App/WeathersList/view/DayForecastWeatherCell.swift (hunk 2 above)
Comment (PiotrChlapinskiBinarapps) [reply Weather App/WeathersList/view/DayForecastWeatherCell.swift]: 👍

File: Weather App/WeathersList/viewModel/CityWeatherViewModel.swift (hunk 3 above)
Comment (PiotrChlapinskiBinarapps) [reply Weather App/WeathersList/viewModel/CityWeatherViewModel.swift]: 👍

File: Weather App/WeathersList/controller/WeatherDetailsViewController.swift (hunk 4 above)
Comment (PiotrChlapinskiBinarapps) [reply Weather App/WeathersList/controller/WeatherDetailsViewController.swift]: 👍

**Review by jarekdominiak — APPROVED** [review]

**Pull request merged**